tests/track/test_track.py

```diff
@@ -0,0 +1,221 @@
+import shutil
+from filecmp import cmpfiles
+from pathlib import Path
+
+import pytest
+
+from OTVision.config import CONFIG
+from OTVision.helpers.files import get_files
+from OTVision.track.track import main as track
+
+SIGMA_L = CONFIG["TRACK"]["IOU"]["SIGMA_L"]
+SIGMA_H = CONFIG["TRACK"]["IOU"]["SIGMA_H"]
+SIGMA_IOU = CONFIG["TRACK"]["IOU"]["SIGMA_IOU"]
+T_MIN = CONFIG["TRACK"]["IOU"]["T_MIN"]
+T_MISS_MAX = CONFIG["TRACK"]["IOU"]["T_MISS_MAX"]
+
+
+def define_and_create_folders(
```

Comment:
Maybe split up setup code to 2 fixtures as in `test_track_dir` and `test_track_tmp_dir`?

Reply:
Done